xmodule/static_content.py

```diff
@@ -123,44 +123,33 @@ def _ensure_dir(directory):
 def _write_styles(selector, output_root, classes, css_attribute, suffix):
     """
     Write the css fragments from all XModules in `classes`
-    into `output_root` as individual files, hashed by the contents to remove
-    duplicates
+    into `output_root` as individual files
     """
     contents = {}
 
-    css_fragments = defaultdict(set)
     for class_ in classes:
         class_css = getattr(class_, css_attribute)()
-        for filetype in ('sass', 'scss', 'css'):
-            for idx, fragment_path in enumerate(class_css.get(filetype, [])):
-                with open(fragment_path, 'rb') as fragment_file:
-                    fragment = fragment_file.read()
-                css_fragments[idx, filetype, fragment].add(class_.__name__)
-    css_imports = defaultdict(set)
-    for (idx, filetype, fragment), classes in sorted(css_fragments.items()):  # lint-amnesty, pylint: disable=redefined-argument-from-local
-        fragment_name = "{idx:0=3d}-{hash}.{type}".format(
-            idx=idx,
-            hash=hashlib.md5(fragment).hexdigest(),
-            type=filetype)
-        # Prepend _ so that sass just includes the files into a single file
-        filename = '_' + fragment_name
-        contents[filename] = fragment
+        fragment_names = []
+        for fragment_path in class_css.get('scss', []):
```

Comment:
Master implementation  just generates scss files with content however this implementation generates empty scss files, with empty scss I mean something like 
```
.xmodule_edit.xmodule_ConditionalBlock {
}
```
I don't know if that is the intention but if this is not the case you can add the following suggestion to avoid that 

```suggestion
       fragment_names = []
       fragment_paths = class_css.get('scss', [])\

        if not fragment_paths:
            continue

        for fragment_path in fragment_paths:
```

Reply:
@andrey-canon Great catch. Fixed.